test/parallel/test-tcp-wrap-listen.js

```diff
@@ -81,26 +81,26 @@ server.onconnection = function(err, client) {
   };
 };
 
-var net = require('net');
+const net = require('net');
 
-var c = net.createConnection(port);
-c.on('connect', function() {
+const c = net.createConnection(port);
+c.on('connect', () => {
```

Comment:
@addaleax given my changes and the availability of `strictEqual` shouldn't this doc https://github.com/nodejs/http2/blob/master/doc/guides/writing_tests.md be updated to always use `const` and `strictEqual`?  I can make a separate PR for this but wanted to know your thoughts.

Thanks